Update custom-server.md (#2225)
keystonejs/keystone: docs/guides/custom-server.md

- Add additional routes
- Setup additional server middleware (`compress`/`brotli`/etc)
- Notify a 3rd party service when the API is ready
- ...

A **Custom Server** can replace the default and act as the entry point to your
application which consumes your [schema definition](/docs/guides/schema.md). A Custom

> Server must handle initializing a http server which correctly executes any given KeystoneJS Apps.

Reviewer: ```suggestion
Server must handle initialising a http server which correctly executes any given KeystoneJS Apps.
```

Author: I noticed, while reading the documentation, that indeed most was en-UK. New PRs will not contain corrections to en-US..

## Custom Server with manual middleware preparation

> For really fine-grained control, a custom server skips calling

Reviewer: ```suggestion
For really fine-grained control, a custom server can skip calling
```

> `keystone.prepare()` in favor of calling an app's `.prepareMiddleware()`

Reviewer: ```suggestion
`keystone.prepare()` in favour of calling an app's `.prepareMiddleware()`
```
